src/pages/Home.js

```diff
@@ -96,11 +96,21 @@
 
     const signInPositionStyle = {
       padding: '1em 1em 0 0',
-      display: 'flex',
       alignItems: 'center',
       position: 'absolute',
-      top: '1rem',
-      right: '1rem'};
+      top: '0',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6963088)
```suggestion
      top: "0",
```